MovieApp/MovieApp/App/Features/MovieSearch/MoviesViewController/ViewModels/Movie.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+struct Movies: Codable {
+    let results: [Movie]
+}
+
+struct Movie: Codable {
+    var imageURL: String
+    var title: String
+    var description: String
+    
+    enum CodingKeys: String, CodingKey {
+        case imageURL = "poster_path"
+        case title = "original_title"
+        case description = "overview"
+    }
+}
```

Comment:
We should never use a model to represent our network model and UI model. Every layer of our application should have his own model. This is a network layer model so this should be called MovieNetworkModel and you should map that model to MovieViewModel which you then use for populating cells and so on

Reply:
Also use cameCase when naming, its imageUrl

Reply:
and watch out for new lines